ipatests/test_integration/test_adtrust_install.py

```diff
@@ -216,3 +220,88 @@ def test_schema_compat_attribute(self):
         value = (r'ipaexternalmember=%deref_r('
                  '"member","ipaexternalmember")')
         assert value in entry_list
+
+
+class TestIpaNisManage(IntegrationTest):
+    """
+    Tests for ipa-nis-manage CLI tool
+    """
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master)
+
+    def test_ipa_nis_manage_enable(self):
+        """
+        This testcase checks if ipa-nis-manage enable
+        command enables plugin on an IPA master
+        """
+        serverid = realm_to_serverid(self.master.domain.realm)
+        dirsrv_service = "dirsrv@%s.service" % serverid
+        console_msg = (
+            "Enabling plugin\n"
+            "This setting will not take effect until you restart Directory Server.\n"
+            "The rpcbind service may need to be started"
+        )
+        tasks.kinit_admin(self.master)
+        result = self.master.run_command(
+            ["ipa-nis-manage", "enable"],
+            stdin_text=self.master.config.admin_password,
+        )
```

Comment:
Would be a good idea to check status after enabling the plugin and see if it is enabled.
`ipa-nis-manage status`